### PR TITLE
Add Mumble Installation

### PR DESCRIPTION
**Pull request by fkaminski**
From Issue #350. 
I only took the texts and photos from the wiki without installing them myself. So perhaps these are not uptodate...

### CODE REVIEW

File: docs/manual/guides/mumble.de.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    - /anleitungen/mumble/
+weight: 50
+---
+
+## Installation
+
+Informationen zu Mumble findest du über [mumble.sourceforge.net](http://mumble.sourceforge.net). Zur 
+[Installation](https://wiki.mumble.info/wiki/Installing_Mumble) musst du dir die passenden Dateien für dein 
+Betriebssystem herunterladen. Du benötigst nur den Mumble-Client. Die Installation der Mumble-Server Anwendung 
+ist nicht notwendig.
```
Comment (fritzmg) [inline docs/manual/guides/mumble.de.md]: The official website is https://www.mumble.info/ - may be link that, instead of sourceforge?

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Comment by fritzmg** [comment]
Since the screenshots are from Windows Vista (?), I think it would be better to make new, current ones.

**Comment by fkaminski** [comment]
> Since the screenshots are from Windows Vista (?), I think it would be better to make new, current ones.

OK, i will install it (win10) and make new screenshots, but i don't know if i can do it this week ...

File: docs/manual/guides/mumble.de.md (hunk 1 above)
Comment (fkaminski) [reply docs/manual/guides/mumble.de.md]: Ok

**Comment by fkaminski** [comment]
> > Since the screenshots are from Windows Vista (?), I think it would be better to make new, current ones.
> 
> OK, i will install it (win10) and make new screenshots, but i don't know if i can do it this week ...

Ok - see last commit

File: docs/manual/guides/mumble.de.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+
+## Mumble einrichten
+
+Nach dem ersten Start von Mumble sollte sich der »Audio Einstellungs-Assistent« öffnen. Alle Einstellungen können 
+nachträglich angepasst werden.
+
+![Mumble Audio Einstellungs-Assistent](/de/guides/images/de/mumble/mumble-audio-assistant.jpg?classes=shadow)
+
+Wenn man den »Weiter« -Button betätigt, kann man auf dem nächsten Screen die bevorzugte Audio-Ein- und Ausgabegeräte auswählen.
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
Wenn man den »Weiter«-Button betätigt, kann man auf dem nächsten Screen die bevorzugte Audio-Ein- und Ausgabegeräte auswählen.
```

File: docs/manual/guides/mumble.de.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    - /de/anleitungen/mumble/
+    - /anleitungen/mumble/
+weight: 50
+---
+
+## Installation
+
+Informationen zu Mumble findest du über [mumble.info](https://www.mumble.info/). Zur 
+[Installation](https://www.mumble.info/downloads/) musst du dir die passenden Dateien für dein 
+Betriebssystem herunterladen. Du benötigst nur den Mumble-Client. Die Installation der Mumble-Server Anwendung
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
Betriebssystem herunterladen. Du benötigst nur den Mumble-Client. Die Installation der Mumble-Server-Anwendung 
```

**Review by fritzmg — CHANGES_REQUESTED** [review]
Im PR befindet sich noch folgende Datei:

```
docs/manual/guides/images/de/mumble-icon-einstellungen.jpg
```

Dieses Bild wird jedoch gar nicht eingebunden und es handelt sich dabei auch um ein veraltetes Bild.

File: docs/manual/guides/mumble.de.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+Informationen zu Mumble findest du über [mumble.info](https://www.mumble.info/). Zur 
+[Installation](https://www.mumble.info/downloads/) musst du dir die passenden Dateien für dein 
+Betriebssystem herunterladen. Du benötigst nur den Mumble-Client. Die Installation der Mumble-Server Anwendung 
+ist nicht notwendig.
+
+
+## Mumble einrichten
+
+Nach dem ersten Start von Mumble sollte sich der »Audio Einstellungs-Assistent« öffnen. Alle Einstellungen können 
+nachträglich angepasst werden.
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
auch nachträglich angepasst werden.
```

File: docs/manual/guides/mumble.de.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+![Mumble Server Verbindung](/de/guides/images/de/mumble/mumble-server-verbinden.jpg?classes=shadow)
+
+Folgende Daten sollte man eingeben:
+
+- **Bezeichnung:** Kann frei gewählt werden, z.B. "CCA-Mumble-Server"
+- **Adresse:** mumble.c-c-a.org
+- **Port:** 62492
+- **Benutzername:** Kann frei gewählt werden
+
+In der Serverliste sollte der "CCA-Mumble-Server" nun unter den Favoriten auftauchen. Diesen Eintrag auswählen
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
In der Serverliste sollte der »CCA-Mumble-Server« nun unter den Favoriten auftauchen. Diesen Eintrag auswählen 
```

File: docs/manual/guides/mumble.de.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+»Audioeingabe« den Punkt »Gerät« (unter »Schnittstelle«) prüfen ob die passende Eingabequelle/Mikrofon ausgewählt ist.
+
+Wenn es unter »Audioausgabe« einen Ausschlag gibt, dieser aber nur im roten Bereich bleibt sollte man die Regler 
+»Stille bis« und »Sprache über« so verschieben, dass der Ausschlag im grünen Bereich ist.
+
+
+### Bedeutung der Bereiche:
+
+- **Gelber Bereich:** Hier wird noch übertragen, falls es zuvor im grünen Bereich war.
+- **Roter Bereich:** Auch wenn hier ein Ausschlag zu sehen ist, wird hiervon nichts übertragen.
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md:146]:
```suggestion
- **Roter Bereich:** Auch wenn hier ein Ausschlag zu sehen ist, wird hiervon nichts übertragen.

## Contao Mumble-Calls

Wann der nächste Contao Mumble-Call stattfindet, erfährst du auf der [Projekt-Website](https://contao.org/de/mumble-calls.html).
```

File: docs/manual/guides/mumble.de.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+
+## Mumble Statusfenster
+
+Herzlichen Glückwunsch, denn wenn man das nachfolgende Fenster sieht, dann ist man im »Mumble«. Wenn man sich auf 
+dem Mumble-Server anmeldet wird man am Anfang immer automatisch in die »Mos Eisley Bar« verschoben. Das ist 
+eine Art virtueller Aufenthaltsraum. Ansonsten ist der »CCA-Mumble-Server« in folgende »Hauptkanäle« (sog. Channel) 
+geteilt: »Galaxie« und »Universum«. 
+
+Diese wiederum in verschiedene Unterchannel. Alle Channel im »Universum« sind für jeden frei zugänglich, außerdem
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
Diese wiederum in verschiedene Subchannel. Alle Channels im »Universum« sind für jeden frei zugänglich, außerdem 
```

File: docs/manual/guides/mumble.de.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+## Mumble Statusfenster
+
+Herzlichen Glückwunsch, denn wenn man das nachfolgende Fenster sieht, dann ist man im »Mumble«. Wenn man sich auf 
+dem Mumble-Server anmeldet wird man am Anfang immer automatisch in die »Mos Eisley Bar« verschoben. Das ist 
+eine Art virtueller Aufenthaltsraum. Ansonsten ist der »CCA-Mumble-Server« in folgende »Hauptkanäle« (sog. Channel) 
+geteilt: »Galaxie« und »Universum«. 
+
+Diese wiederum in verschiedene Unterchannel. Alle Channel im »Universum« sind für jeden frei zugänglich, außerdem 
+in der »Galaxy« die Channel »Galaktische Spiele«, »Galaktischer Salat« und »Galaktischer Senat« mit »offener Senat«.
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
sind in der »Galaxy« die Channels »Galaktische Spiele«, »Galaktischer Salat« und »Galaktischer Senat« mit »offener Senat« für euch zugänglich.
```

File: docs/manual/guides/mumble.de.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+
+Herzlichen Glückwunsch, denn wenn man das nachfolgende Fenster sieht, dann ist man im »Mumble«. Wenn man sich auf 
+dem Mumble-Server anmeldet wird man am Anfang immer automatisch in die »Mos Eisley Bar« verschoben. Das ist 
+eine Art virtueller Aufenthaltsraum. Ansonsten ist der »CCA-Mumble-Server« in folgende »Hauptkanäle« (sog. Channel) 
+geteilt: »Galaxie« und »Universum«. 
+
+Diese wiederum in verschiedene Unterchannel. Alle Channel im »Universum« sind für jeden frei zugänglich, außerdem 
+in der »Galaxy« die Channel »Galaktische Spiele«, »Galaktischer Salat« und »Galaktischer Senat« mit »offener Senat«. 
+
+In den Channel »geschlossener Senat« (in »Galaktischer Senat«) dürfen nur Administratoren. Diese Administratoren
```
Comment (netzarbeiter) [inline docs/manual/guides/mumble.de.md]:
```suggestion
Im Channel »geschlossener Senat« (in »Galaktischer Senat«) dürfen nur Administratoren. Diese Administratoren 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Danke, Franko

**Pull request merged**